xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/displayers/BaseDisplayer.vue

```diff
@@ -24,57 +24,60 @@
   specific displayer that pass to it a `viewer` and `editor` slot.
   In that way, specific displayers only care about implementing
   the formatting of its content (view mode and edit mode)
-  instead of reimplemting the whole displayer logic each time.
+  instead of reimplementing the whole displayer logic each time.
 -->
 <template>
-  <div
-    :class="isView ? 'view' : 'edit'"
-    @dblclick="edit"
-    @keypress.self.enter="edit"
-    :tabindex="isEditable ? 0 : ''"
-  >
+  <div :class="{view: isView, edit: !isView, editing: isView}" ref="displayerRoot">
```

Comment:
What does ``div.view.editing`` mean? Why is this ``editing`` CSS class needed on view mode?